apps/contracts/contracts/Zengo.sol

```diff
@@ -100,7 +116,10 @@ contract ZengoDAO is Constants, ZengoStorage, GStates, PermissionsEnumerable, Co
         string memory _evidenceDescription,
         string memory _evidenceUri,
         string memory _proposalType,
-        uint256 _proposalEvidenceTimestamp
+        uint256 _proposalEvidenceTimestamp,
+        string memory _streetAddress,
```

Comment:
@Markkos89 Can you group all the dataType declaration? It will use less gas and is a standard practice